dev/com.ibm.ws.classloading/resources/OSGI-INF/l10n/library.properties

```diff
@@ -45,26 +45,23 @@ library.api.thirdparty=Third party interfaces
 library.api.thirdparty.desc=Interfaces defined by third party vendors
 library.folder.ref=Folder
 library.folder.ref$Ref=Folder reference
-library.folder.ref.desc=Id of referenced folder
 library.file.ref=File
 library.file.ref$Ref=File reference
-library.file.ref.desc=Id of referenced File
 library.path.ref=Path
 library.path.ref$Ref=Path reference
-library.path.ref.desc=Id of referenced path
 #File element
 file=File
-file.desc=Add a JAR or ZIP file to include in the class path for this library
+file.desc=Add a JAR or ZIP file to include in the class path for this library. Classes and resources contained in the specified JAR or ZIP are available to load for this library.
 file.name=Name
 file.name.desc=The path to a file can be a fully qualified path or a relative path. Relative paths are relative to the value of the ${server.config.dir} property.
 # Folder element
 folder.name=Folder
-folder.name.desc=Add a folder to include in the class path for this library
+folder.name.desc=Add a folder to include in the class path for this library. Classes and resources contained in the specified folder and its sub-folders are available to load for this library. JAR and ZIP files contained in the folder are not added to the class path for this library.
 folder.dir=Folder
 folder.dir.desc=The path to a folder can be a fully qualified path or a relative path. Relative paths are relative to the value of  the ${server.config.dir} property.
 #path element
 path=Path
-path.desc=Add a JAR file, ZIP file or folder to include in the class path for this library
+path.desc=Add a JAR file, ZIP file or folder to include in the class path for this library. If the specified path is a JAR or ZIP file then classes and resources contained in the specified JAR or ZIP file are available to load for this library. If the specified path is a folder then classes and resources contained in the specified folder and its sub-folders are available to load for this library, and JAR or ZIP files contained in the folder are not added to the class path for this library.
```

Comment:
Add "that are" before contained and subfolders should be one word. Also, the second sentence should be broken up into two sentences.

So change to: "If the specified path is a JAR or ZIP file, then classes and resources that are contained in the specified JAR or ZIP file are available to load for this library. If the specified path is a folder, then classes and resources that are contained in the specified folder and its subfolders are available to load for this library. JAR and ZIP files contained in the specified folder are not added to the class path for this library."

Reply:
I also applied these suggestions to `file.desc`